CHANGES.md

```diff
@@ -1,3 +1,27 @@
+## edge-19.6.2
+
+* CLI
+  * Added the `--linkerd-cni-enabled` flag to the `install` subcommands so that
+    `NET_ADMIN` capability is omitted from the control plane's PSP
+* Controller
+  * Fixed security context values so that injection does not fail on
+    restricted PSPs (thanks @codeman9!)
+  * Fixed webhook failure policy in order to avoid timing issues in HA mode
+  * Introduced control plane's PSP and RBAC resources into Helm templates;
+    these policies are only in effect if the PSP admission controller is
+    enabled
+  * Fixed MWC namespace value so that when installing multiple control planes,
+    there is a unique configuration for each one
```

Comment:
User action is required here, if they are using the `linkerd upgrade` command to upgrade the latest edge. After the upgrade, they will see two mutating webhook configurations:

```
k get mutatingwebhookconfigurations.admissionregistration.k8s.io
NAME                                    CREATED AT
linkerd-proxy-injector-webhook-config   2019-06-13T22:22:55Z
linkerd-linkerd-proxy-injector-webhook-config 2019-06-13T22:22:55Z

```
They will need to manually remove the `linkerd-proxy-injector-webhook-config` mwc. This isn't relevant if they are doing a fresh installation of the edge release.

Reply:
Is this included in docs? I think this information is better conveyed through docs rather than the release notes

Reply:
sgtm. yeah, probably not needed for an edge release.